manifests/1.14/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -1,13 +1,10 @@
-kind: StatefulSet
+kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: vsphere-csi-controller
   namespace: kube-system
 spec:
-  serviceName: vsphere-csi-controller
   replicas: 1
-  updateStrategy:
```

Comment:
`updateStrategy` -> `RollingUpdate` will be useful later for upgrade. Can you keep that?